<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMovie</class>
 <widget class="QDialog" name="AddMovie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>560</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>741</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameLineEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Director</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorLineEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="descLineEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Link</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="linkLineEdit_2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Year</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="yearSpinBox_2">
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="genreComboBox_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
